userspace/libscap/plugin_info.h

```diff
@@ -151,8 +131,7 @@ typedef struct ss_plugin_event
 //   If the field is a list type, then res_len can must be any value from 0 to N, depending
 //   on how many values can be extracted from a given event.
 //   Setting res_len to 0 means that no value of this field can be extracted from a given event.
-
-
+//
```

Comment:
I think we can remove this or we can add the corresponding `//` at the beginning of the comment